extensions/amp-list/0.1/amp-list.js

```diff
@@ -446,6 +468,43 @@ export class AmpList extends AMP.BaseElement {
       });
   }
 
+  /**
+   * Gets the json from an amp-script uri.
+   *
+   * @param {string} src
+   * @return {Promise<!JsonObject>}
+   * @private
+   */
+  getAmpScriptJson_(src) {
+    return Services.scriptForDocOrNull(this.element)
+      .then((ampScript) => {
+        userAssert(
+          ampScript,
+          '"amp-script:" URLs require amp-script to be installed.'
+        );
+        userAssert(
+          !this.ssrTemplateHelper_.isEnabled(),
+          '[amp-list]: "amp-script" URIs cannot be used in SSR mode.'
+        );
+
+        const args = src.slice(AMP_SCRIPT_URI_SCHEME.length).split('.');
+        userAssert(
+          args.length === 2,
+          '[amp-list]: "amp-script" URIs must be of the format "scriptId.functionIdentifier".'
+        );
```

Comment:
Should we check that id and fn have non-zero length?

Reply:
sure